src/extensions/jana/JChainMultifactoryT.h

```diff
@@ -19,11 +19,15 @@
 #include <string>
 #include <vector>
 
+#include <algorithms/property.h>
 #include "datamodel_glue.h"
 #include <JANA/JMultifactory.h>
 #include "algorithms/interfaces/WithPodConfig.h"
 #include "extensions/jana/JChainFactoryT.h"  // Just to pull in struct NoConfig
 
+namespace eicrecon {
+    using ConfigMap = std::map<std::string, algorithms::PropertyValue>;
```

Comment:
Shoulda used a string_view as these are all read only.
```suggestion
    using ConfigMap = std::map<std::string_view, algorithms::PropertyValue>;
```